apps/hash-frontend/src/pages/actions.page/draft-entities/draft-entities-context-bar.tsx

```diff
@@ -107,71 +107,68 @@ export const DraftEntitiesContextBar: FunctionComponent<{
+            textTransform: "uppercase",
+          }}
+        >
+          {selectedDraftEntityIds.length} drafts{" "}
+          <Box
+            component="span"
             sx={{
-              svg: {
-                width: 18,
-                height: 18,
-              },
-            }}
-          />
-
-          <Typography
-            sx={{
-              display: "block",
-              fontSize: 12,
-              color: ({ palette }) => palette.common.black,
-              fontWeight: 600,
-              textTransform: "uppercase",
+              color: ({ palette }) => palette.gray[70],
             }}
           >
-            {selectedDraftEntityIds.length} drafts{" "}
-            <Box
-              component="span"
-              sx={{
-                color: ({ palette }) => palette.gray[70],
-              }}
-            >
-              selected
-              {hasSelectedAllDisplayedDraftEntities &&
-              hasPartiallySelectedMatchingDraftEntities
-                ? " (all on this page)"
-                : ""}
-              .
-            </Box>
-          </Typography>
-          {hasPartiallySelectedDisplayedDraftEntities ? (
-            <SelectAdditionalButton
-              variant="secondary_quiet"
-              onClick={handleSelectAllDisplayedDraftEntitiesClick}
-              endIcon={<CheckRegularIcon />}
-            >
-              Select all {displayedDraftEntities.length} on this page
-            </SelectAdditionalButton>
-          ) : hasPartiallySelectedMatchingDraftEntities ? (
-            <SelectAdditionalButton
-              variant="secondary_quiet"
-              onClick={handleSelectAllMatchingDraftEntitiesClick}
-              endIcon={<CheckRegularIcon />}
-            >
-              Select all {matchingDraftEntities.length}{" "}
-              {isDefaultFilterState ? "drafts" : "matching current filters"}
-            </SelectAdditionalButton>
-          ) : null}
-        </Box>
-      </Fade>
+            selected
+            {hasSelectedAllDisplayedDraftEntities &&
+            hasPartiallySelectedMatchingDraftEntities
+              ? " (all on this page)"
+              : ""}
+            .
+          </Box>
+        </Typography>
+        {hasPartiallySelectedDisplayedDraftEntities ? (
+          <SelectAdditionalButton
+            variant="secondary_quiet"
+            onClick={handleSelectAllDisplayedDraftEntitiesClick}
+            endIcon={<CheckRegularIcon />}
+          >
+            Select all {displayedDraftEntities.length} on this page
+          </SelectAdditionalButton>
+        ) : hasPartiallySelectedMatchingDraftEntities ? (
+          <SelectAdditionalButton
+            variant="secondary_quiet"
+            onClick={handleSelectAllMatchingDraftEntitiesClick}
+            endIcon={<CheckRegularIcon />}
+          >
+            Select all {matchingDraftEntities.length}{" "}
+            {isDefaultFilterState ? "drafts" : "matching current filters"}
```

Comment:
```suggestion
            {isDefaultFilterState ? "items" : "matching current filters"}
```

Reply:
Addressed in https://github.com/hashintel/hash/pull/4656/commits/323ede31d0e102018e0dcbea0d0cfaf479362f17